docs/ref/class-based-views/base.txt

```diff
@@ -65,6 +65,11 @@ View
 
             response = MyView.as_view()(request)
 
+        .. versionadded:: 1.9
```

Comment:
These notes should be "standalone" per https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features (currently we'd have to deindent them when removing the annotations)